unabridged-works-volume-1/original/03-way-of-life.adoc

```diff
@@ -157,15 +158,16 @@ shall in the end lose all, and yet not gain this either;
 therefore be no longer wise in the eye of flesh, or according to what man calleth wisdom;
 but be TRULY wise.
 
-=== SOME POSITIONS Concerning the Apostasy from the Christian Spirit and Life
+=== Some Positions Concerning the Apostasy from the Christian Spirit and Life
```

Comment:
Here's on where i wasn't sure if I should use the [.old-style] as well?

Reply:
I think this is fine. 👍